README.md

```diff
@@ -132,6 +132,11 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
 This should resolve native library loading issues.
 
 ## Release Notes
+
+### Release 2.0.0 (27 May 2020)
+* Updated docker scripts to incorporate the changes in producer SDK
+* Provision of CachedInfoMultiAuthServiceCallbacks callback to use different credential providers for different streams in one client with existing stream information
```

Comment:
Could you perhaps add a little more context like:

Provision of 'CachedInfoMultiAuthServiceCallbacks' callbacks which implements caching layer and per-stream auth which is useful in scenarios where a single client object is used for multiple streams that are intended for different accounts. This is useful for service proxy type of scenarios.


Please reword this as you see fit

Reply:
Sure!